### PR TITLE
removing download error from EarthExplorer

### LINKED ISSUE #140: download_image error using EarthExplorer
**Describe the bug**
Error produced while downloading Landsat image from EarthExplorer 
```python
 request() got an unexpected keyword argument 'datasetName'
```
**To Reproduce**
Removing the lines [302](https://github.com/dlr-eoc/ukis-pysat/blob/master/ukis_pysat/data.py#L302) to [310](https://github.com/dlr-eoc/ukis-pysat/blob/master/ukis_pysat/data.py#L310) from data.py

**Expected behavior**
the required Landsat image according to the product_uuid should have been downloaded

### PR DESCRIPTION
**Pull request by shiwakotisurendra**
closes #140

### CODE REVIEW

File: ukis_pysat/data.py [hunk 1]
```diff
@@ -299,15 +299,6 @@ def download_image(self, product_uuid, target_dir):
 
         elif self.src == Datahub.EarthExplorer:
             # query EarthExplorer for srcid of product
```
Comment (fwfichtner) [inline ukis_pysat/data.py:301]: Can you delete this line again?

**Review by fwfichtner — CHANGES_REQUESTED** [review]
did you test this? This looks like `product_srcid` should be renamed to `product_uuid` as well, because it does not exist in this case.

Also our comment in line `305` is wrong, it should be `GCS` instead of `AWS`.

Please update CHANGELOG as well.

**Comment by shiwakotisurendra** [comment]
> did you test this? This looks like `product_srcid` should be renamed to `product_uuid` as well, because it does not exist in this case.
> 
> Also our comment in line `305` is wrong, it should be `GCS` instead of `AWS`.
> 
> Please update CHANGELOG as well.

yes i did this already and also tested , so it is working already .

File: ukis_pysat/data.py (hunk 1 above)
Comment (shiwakotisurendra) [reply ukis_pysat/data.py]: done

File: CHANGELOG.rst [hunk 2]
```diff
@@ -15,8 +15,9 @@ something was changed
 
 [master] (2021-**-**)
 ---------------------
+- ``data``: fixed download error from EarthExplorer #140
 
-[1.2.0]] (2021-04-07)
+[1.2.0]] (2021-04-07)s
```
Comment (fwfichtner) [inline CHANGELOG.rst:20]: Can you delete the `s`

**Review by fwfichtner — APPROVED** [review]
Guess that should work

File: CHANGELOG.rst (hunk 2 above)
Comment (shiwakotisurendra) [reply CHANGELOG.rst]: done

**Review by fwfichtner — APPROVED** [review]

**Pull request merged**